.env.example

```diff
@@ -1,3 +1,30 @@
+# Choose one of the following environment configuration: local (development), goerli (staging), mainnet (production)
+# This will automatically set other required environment variables to point to choosen configuration.
+
+# Development: point to your local ganache chain. Usit it when running Ocean stack locally with Barge.
+NODE_ENV = "development"
+
+# Staging: point to Goerli network.
+NODE_ENV = "staging"
+
+# Production: point to Ethereum Mainnet.
+NODE_ENV = "production"
+
+
+
+
+# You can enable or disable rollup livereload to start a server and reload the page on saving changes.
+# This is usualy enabled on local and disable on production.
+START_RELOAD_SERVER = true
+
+
+
+
+# You can also set your custom environment variables by adding a different value for NODE_ENV then the specified ones above, and manually setting the other variables.
+
+# set one of Ocean subgraphs from the available subgrapsh list from https://github.com/oceanprotocol/ocean-subgraph.
```

Comment:
We should keep the examples in the comments like we have currently. That makes it easier for people to choose which subgraph they want:
```suggestion

# choose one of the following local subgraph (dev), goerli (staging), and mainnet (prod)
# Dev: http://172.15.0.15:8000/subgraphs/name/oceanprotocol/ocean-subgraph
# Staging: https://v4.subgraph.goerli.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph
# Production: https://v4.subgraph.mainnet.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph
```

Reply:
The ideal would be for user to set one of the mentions for `NODE_ENV` and then they won't need to configure anything.
If they want something else there is an example at the end of the file with all the variables he needs to set. The subgraphs are not listed but we give one example and point user to all our subgraphs so he can choose one from there.

Reply:
I will comment not necessary configs

Reply:
It doesn't say anywhere in the subgraph readme the address of the subgraph in barge

Reply:
That's right, I just checked, looks like subgraphs readme should be updated